don't hardcode $CMAKE_CXX_FLAGS
author: Kenneth Hoste (HPC-UGent)
-- sailfish-0.8.0/CMakeLists.txt.orig	2015-12-07 12:05:52.558531939 +0100
+++ sailfish-0.8.0/CMakeLists.txt	2015-12-07 13:10:42.383652429 +0100
@@ -25,7 +25,7 @@
 
 ## Set the standard required compile flags
 # Nov 18th --- removed -DHAVE_CONFIG_H
-set (CMAKE_CXX_FLAGS "-pthread -funroll-loops -fPIC -fomit-frame-pointer -Ofast -DHAVE_ANSI_TERM -DHAVE_SSTREAM -DRAPMAP_SALMON_SUPPORT -Wall -std=c++11 -Wreturn-type -Werror=return-type")
+set (CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -pthread -DHAVE_ANSI_TERM -DHAVE_SSTREAM -DRAPMAP_SALMON_SUPPORT -Wall -std=c++11 -Wreturn-type -Werror=return-type")
 
 ##
 # OSX is strange (some might say, stupid in this regard).  Deal with it's quirkines here.
